lib/rules/lint-no-bare-strings.js

```diff
@@ -141,6 +148,13 @@ module.exports = class LogStaticStrings extends Rule {
   }
 
   visitor() {
+    if (
+      this.config.ignoreDummyTestApp &&
+      this.templateEnvironmentData.moduleName.startsWith('tests/dummy/app/templates/')
```

Comment:
This doesn't work when used via ember-cli-template-lint (the module name would be `dummy/templates/` not `tests/dummy/app/templates/`)